reporters/plain_reporter.py

```diff
@@ -11,9 +12,34 @@ def handle_message(self, msg):
         self._messages.append(msg)
 
     def print_message_ids(self):
+        # Sort the messages.
+        self.sort_messages()
+        for msg in self._messages:
+            code = msg.msg_id
+            print(code, '({}) {}\n    {}'.format(msg.symbol, msg.obj, msg.msg))
+
+    def sort_messages(self):
         # Sort the messages by their type.
         self._messages.sort(key=lambda s: s[0])
 
-        for msg in self._messages:
-            code = msg.msg_id
-            print(code, '({})\n    {}'.format(msg.symbol, msg.msg))
+        i = 0
+        while i < len(self._messages):
+            current_id = self._messages[i].msg_id
+            count = 1
+            messages = []
+            while i + 1 < len(self._messages) and self._messages[i + 1].msg_id == current_id:
+                count += 1
+                if len(messages) <= 5:
+                    messages.append(self._messages[i+1].msg)
```

Comment:
Put spaces around `+`.

When I give you feedback about style, please check over all of your code changes so that I don't need to point out the same error at every location.